p2p/peering.go

```diff
@@ -119,6 +119,37 @@ type LocalPeer struct {
 	rpMu   sync.Mutex
 }
 
+type SnapShot struct {
+	Id         uint64
+	Services   wire.ServiceFlag
+	InitHeight int32
+	Raddr      string
```

Comment:
"raddr" is shorthand, for external api, name this Addr (the type is already specific to remote peers).

Reply:
fixed